sections/featured-blog.liquid

```diff
@@ -178,7 +218,7 @@
           "label": "t:sections.all.heading_size.options__3.label"
         }
       ],
-      "default": "h1",
+      "default": "h2",
```

Comment:
Why did we change this?

Reply:
I noticed that all of the headings for the sections on the home page were h2 and having the default for blogs posts be h1 was slightly odd to the eye. But i can change it back. I can also check with ux to see what they would prefer :)

Reply:
yes, let's check with UX 👍

Reply:
I reverted it back to be defaulted to h1 and we can always make a separate PR for it  :)